db_json_util.py

```diff
@@ -74,6 +76,35 @@ def main() -> None:
                     else:
                         print(response.text)
 
+# export data using the API
+        if args.export_data:
+            if not os.path.exists('json_exports'):
+                os.makedirs('json_exports')
+            if args.json_chains:
+                url_format = args.url + '/all/chains'
+                response = requests.get(url_format)
+                if response.status_code == 200:
+                    data_chains = response.json()
+                    export_name = "json_exports/chains_exported.json"
+                    with open(export_name, 'w', encoding='utf-8') as f:
+                        json.dump(data_chains, f, indent=4)
+                        export_name = "chains_exported.json"
+                    print(f'exported chains to {export_name}')
+                else:
+                    print(response.text)
+            if args.json_rpc_urls:
+                url_format = args.url + '/all/rpc_urls'
+                response = requests.get(url_format)
+                if response.status_code == 200:
+                    data_urls = response.json()
+                    export_name = "json_exports/rpc_urls_exported.json"
```

Comment:
Same here, but use `path_urls`. Note that `path_urls` is a `Path` object, it can be used directly in the `open()` function but if you for some reason want the path as a string you'll need to use `str(path_urls)`.